mesonbuild/backend/ninjabackend.py

```diff
@@ -175,11 +175,16 @@ def detect_vs_dep_prefix(self, tempfilename):
                               stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         (stdo, _) = pc.communicate()
 
+        # We want to match 'Note: including file: ' in the line
+        # 'Note: including file: d:\MyDir\include\stdio.h', however
+        # different locales have diffent messages with different
+        # number of colons. Match up to the the drive name 'd:\'
+        matchre = re.compile(b"^(.*\s)[a-zA-Z]:\\\\.*stdio.h$")
```

Comment:
`\s` is a valid regex pattern:

```
For Unicode (str) patterns:
    Matches Unicode whitespace characters (which includes [ \t\n\r\f\v], and also many other characters
```

And this is already a byte string, how is `b''` different from `rb''`?